shared/django_apps/codecov_auth/models.py

```diff
@@ -87,6 +87,7 @@ class CustomerIntent(models.TextChoices):
     terms_agreement = models.BooleanField(null=True, default=False, blank=True)
     terms_agreement_at = DateTimeWithoutTZField(null=True, blank=True)
     customer_intent = models.TextField(choices=CustomerIntent.choices, null=True)
+    email_opt_in = models.BooleanField(null=True, default=False, blank=True)
```

Comment:
Hi Rola! You don't need to set `blank` here.

Reply:
also, do you want to make this nullable - does `None` have a business logic meaning in this field?

Reply:
i guess not, will remove